db/migrate/20221212061802_enable_multi_searchable.rb

```diff
@@ -0,0 +1,12 @@
+class EnableMultiSearchable < ActiveRecord::Migration[6.1]
+  def up
+    Contact.rebuild_pg_search_documents
+    PgSearch::Multisearch.rebuild(Conversation)
+    PgSearch::Multisearch.rebuild(Message)
+    execute 'CREATE EXTENSION IF NOT EXISTS pg_trgm;'
+  end
```

Comment:
- We need to estimate the time of this migration while executing on prod. Since the message table could be quite big, I am concerned about the time this migration may take, so we need to test this thoroughly. 

- I think it's better to do it as a sidekiq job than directly inline. 

- we need to ensure the pg_trgm extension is available and working on the cloud platforms which we support like ubuntu, AWS, GCP etc.

Reply:
@sojan-official This Makes sense; let me move it to sidekiq job.

Reply:
pg_trgm available:

GCP extension https://cloud.google.com/alloydb/docs/reference/extensions
AWS extension https://docs.aws.amazon.com/AmazonRDS/latest/PostgreSQLReleaseNotes/postgresql-extensions.html
Ubuntu apt-get installs are available with the pg_trgm extension.